Converters/Rhino/Speckle.Converters.RhinoShared/ToHost/Raw/FlatPointListToHostConverter.cs

```diff
@@ -20,20 +25,18 @@ public class FlatPointListToHostConverter : ITypedConverter<IReadOnlyList<double
   /// with the numbers being coordinates of each point in the format {x1, y1, z1, x2, y2, z2, ..., xN, yN, zN}
   /// </remarks>
   /// <exception cref="SpeckleException">Throws when the input list count is not a multiple of 3.</exception>
-  public Point3dList Convert(IReadOnlyList<double> target)
+  public Point3dList Convert(IReadOnlyList<double> target) => new(ConvertToEnum(target));
+
+  public IEnumerable<RG.Point3d> ConvertToEnum(IReadOnlyList<double> target)
```

Comment:
what is the benefit of convertin enum here? need more inline docs to remember later the reasoning

Reply:
It's avoiding temporary collections